openedx/features/enterprise_support/tests/test_api.py

```diff
@@ -26,6 +30,9 @@
 from student.tests.factories import UserFactory
 
 
+ENTERPRISE_SERVICE_WORKER_USERNAME = 'enterprise_worker'
```

Comment:
Should we declare this here or get it from `settings`?

Reply:
Thanks for pointing this out. I will use this value from settings.